packages/maker.js/src/core/maker.ts

```diff
@@ -130,8 +130,9 @@ namespace MakerJs {
         if (n % 1 === 0) return n;
 
         const temp = 1 / accuracy;
+        const eps = Number.EPSILON || Math.pow(2, -52);
```

Comment:
can we move this to the outer scope?

Reply:
As an export ?

Reply:
no export yet. I imagine it may be promoted when needed though.

Reply:
It will be in the scope of the namespace. So it won't evaluate the conditional `||` every call.

Reply:
Done